data/earnings.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
     const now = new Date();
     // now = new Date(now.getFullYear(), now.getMonth());
 
-    for (let i = 0; i < 7; i++) {
+    for (let i = 0; i < 7 * 10; i++) {
```

Comment:
Since I was also working Influx as one of my DB. Leslie wanted me to have 1M primary records, but that meant I needed to up the count for the number of earnings in each record.